### ISSUE TITLE
Deselect multiple files after copy

### ISSUE THREAD

**Issue by joshas** [fman-community/fman#112]
After copying multiple files, source files are still selected with no indication about success or failure of copy operation. At least files that were copied should be deselected. Having a non-blocking notification about success would be great. E.g. Total Commander makes a "beep" sound after long operations.

**Comment by mherrmann** [comment]
I agree it's very basic at the moment, but it does say which file is currently being copied in the status bar (`Copying abc.txt...`). When it's done copying, it says `Ready.`. A "beep" sound should be a separate issue, in my opinion.

I'll change the title of this present issue from "Multiple files are not deselected after copy" to "Deselect multiple files after copy". In my view, it's an enhancement, not a bug.

**Comment by macosxguru** [comment]
This gets handled with your new **Deselect** now command, right?

macosxguru

**Comment by mherrmann** [comment]
Mh, it's not quite the same. Total Commander deselects files as they are being processed. So it gives a visual indication what has already been completed during the operation. Also I suppose it would be useful when something goes wrong during the operation. For instance, when you copy and run out of disk space, TC's implementation would (presumably) then only still have the files selected which remained to be copied, so you can maybe free up some disk space and then continue with only the still selected files. But you are right in that **Deselect** easily fixes the "annoyance" of still having files selected after a successful operation.

**Comment by macosxguru** [comment]
Ahhh. The Total Commander behavior is nice.

macosxguru

**Comment by mherrmann** [comment]
Yes, it also has another behaviour that's really nice: #41. Would love to soon have this in fman as well.

**Comment by macosxguru** [comment]
Michael, you better get to work. ;)

macosxguru

**Comment by mherrmann** [comment]
Yes, I'm a slacker I know ;) It is 6:33 am for me and there are _still_ 114 open issues. At least, I'm just releasing fman 0.3.7 with a fix for #102.

**Comment by macosxguru** [comment]
Don't stress. There are no time limits on the 114 open issues. The crashing bugs though need attention. So good work.

macosxguru

**Comment by mrexodia** [comment]
Would be really good to have this feature. I'm so used to it from Total Commander and I noticed it's very disruptive to keep the selection. My scenario:

1. Filter some files with "table" (it shows various xxxTable.cpp xxxTable.h files)
2. Copy them to the other pane (my project directory)
3. Switch to the IDE and use the copied files
4. 5/10 minutes later switch to fman to copy some other missing files
5. Remove the filter, start filtering for some other files
6. The old selection stays and I now have to manually deselect them or switch out of the folder

Would there be a way for me to implement this behaviour as a plugin, perhaps I can hook into the copy/move command or would I have to reimplement the command to also deselect the files in the pane?

**Comment by mherrmann** [comment]
Hi, you can clear the selection with <kbd>Ctrl</kbd>+<kbd>D</kbd> (or <kbd>Cmd</kbd>+<kbd>D</kbd> on Mac).
